bioimageio/core/build_spec/build_model.py

```diff
@@ -258,6 +266,160 @@ def _get_dependencies(dependencies, root):
     return model_spec.raw_nodes.Dependencies(manager=manager, file=_process_uri(path, root))
 
 
+def _get_deepimagej_macro(name, kwargs, export_folder):
+
+    # macros available in deepimagej
+    assert name in ("scale_linear", "scale_range", "zero_mean_unit_variance")
+    # TODO deep imagej has a binarize macro but I have no idea what it is doing
+    # assert name in ("binarize", "scale_linear", "scale_range", "zero_mean_unit_variance")
+
+    if name == "scale_linear":
+        macro = "scale_linear.ijm"
+        replace = {"gain": kwargs["gain"], "offset": kwargs["offset"]}
+
+    elif name == "scale_range":
+        macro = "per_sample_scale_range.ijm"
+        replace = {"min_precentile": kwargs["min_percentile"], "max_percentile": kwargs["max_percentile"]}
+
+    elif name == "zero_mean_unit_variance":
+        mode = kwargs["mode"]
+        if mode == "fixed":
+            macro = "fixed_zero_mean_unit_variance.ijm"
+            replace = {"paramMean": kwargs["mean"], "paramStd": kwargs["std"]}
+        else:
+            macro = "zero_mean_unit_variance.ijm"
+            replace = {}
+
+    # elif name == "binarize":
+    #     macro = "binarize.ijm"
+    #     replace = {"", kwargs["threshold"]}
+
+    macro = f"{name}.ijm"
+    url = f"https://raw.githubusercontent.com/deepimagej/imagej-macros/master/bioimage.io/{macro}"
+
+    path = os.path.join(export_folder, macro)
+    # TODO do we use requests?
+    with requests.get(url, stream=True) as r:
```

Comment:
let me update https://github.com/bioimage-io/core-bioimage-io-python/blob/74e0f4beafabdb5a43df0d082dd3731c7f968441/bioimageio/core/resource_io/utils.py#L267 to accept `local_path` to get a nice tqdm pbar and then use it here...